src/components/radio/radio_group.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
Yeah, that sidenav should also be a directive AFAICT (unless there's some css reason it's like that, @hansl?).

You use `@Directive` when there _is_ no template, so the content is just where it naturally resides in the tree.